### ISSUE TITLE
java.lang.RuntimeException: android.os.TransactionTooLargeException: data parcel size 625168 bytes

### ISSUE THREAD

**Issue by ProfessorVenkman** [Electron-Cash/Electron-Cash#1085]
Crash Report
============

This crash report was reported through the automatic crash reporting system 🤖

Traceback
--------------

```Python traceback
	at android.app.ActivityThread$StopInfo.run(ActivityThread.java:4156)
	at android.os.Handler.handleCallback(Handler.java:789)
	at android.os.Handler.dispatchMessage(Handler.java:98)
	at android.os.Looper.loop(Looper.java:164)
	at android.app.ActivityThread.main(ActivityThread.java:6944)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:327)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1374)
Caused by: android.os.TransactionTooLargeException: data parcel size 625168 bytes
	at android.os.BinderProxy.transactNative(Native Method)
	at android.os.BinderProxy.transact(Binder.java:761)
	at android.app.IActivityManager$Stub$Proxy.activityStopped(IActivityManager.java:5165)
	at android.app.ActivityThread$StopInfo.run(ActivityThread.java:4148)
	... 7 more
java.lang.RuntimeException: android.os.TransactionTooLargeException: data parcel size 625168 bytes
```

Reporter
------------

This issue was reported by 1 user(s):

| Electron Cash Version  | Python Version | Operating System  | Wallet Type  | Locale |
|---|---|---|---|---|
| 3.3.4-1  | 3.6.5 (default, Dec  7 2018, 13:59:54)  [GCC 4.9] | Android 8.0.0 on samsung SM-A530W (jackpotltecan) | unknown | en_CA | 

Additional Information
------------------------

The reporting user(s) did not provide additional information.

**Comment by mhsmith** [comment]
I've seen this happen before in a different app which used the same console activity code, when there was so much text in the console that the system refused to save it. 

Since the console is only there for developer convenience, I'm not going to look into this just now, unless there's some indication that the crash can happen to people who aren't using the console.

**Comment by cculianu** [comment]
Ahl.. hah. I see. Yeah let's close it.

**Comment by cculianu** [comment]
Dupe of #1285

**Comment by ProfessorVenkman** [comment]
Good afternoon @cculianu,

I just received another crash report related to this issue. The crash occured on Electron Cash 3.3.5-1.
I'm not sure which versions of Electron Cash include the fix but this is the first report from anything
newer than 3.3.4-1 since you closed the issue.

Could you please check if this issue really is resolved? Here is the traceback that I just collected:

```Python traceback
	at android.app.ActivityThread$StopInfo.run(ActivityThread.java:4156)
	at android.os.Handler.handleCallback(Handler.java:789)
	at android.os.Handler.dispatchMessage(Handler.java:98)
	at android.os.Looper.loop(Looper.java:164)
	at android.app.ActivityThread.main(ActivityThread.java:6942)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:327)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1374)
Caused by: android.os.TransactionTooLargeException: data parcel size 895824 bytes
	at android.os.BinderProxy.transactNative(Native Method)
	at android.os.BinderProxy.transact(Binder.java:761)
	at android.app.IActivityManager$Stub$Proxy.activityStopped(IActivityManager.java:5165)
	at android.app.ActivityThread$StopInfo.run(ActivityThread.java:4148)
	... 7 more
java.lang.RuntimeException: android.os.TransactionTooLargeException: data parcel size 895824 bytes
```


~ _With robotic wishes_

**Comment by mhsmith** [comment]
The fix will be in version 4.0.9, which we are currently trying to get published on Google Play.
